app/models/case_request.rb

```diff
@@ -5,10 +5,10 @@ class CaseRequest
     :confirmation_code,
     :fees
 
-  def initialize(opts = {})
-    @case_reference = opts[:case_reference]
-    @confirmation_code = opts[:confirmation_code]
-    @fees = []
+  def initialize(case_reference, confirmation_code)
+    @case_reference    = case_reference
+    @confirmation_code = confirmation_code
+    @fees              = []
```

Comment:
Again, sorry - force of habit (and I've got it bound to a keystroke, too). I'll change this to match the house style.